Extensions/ExtendedMath-header.json

```diff
@@ -0,0 +1,13 @@
+{
+  "shortDescription": "A general extention that adds additional math functions and constants expressions.",
+  "extensionNamespace": "",
+  "fullName": "Extended math support",
+  "name": "ExtendedMath",
+  "version": "1.0.0",
+  "url": "Extensions/ExtendedMath.json",
+  "headerUrl": "Extensions/ExtendedMath-header.json",
+  "tags": "math, constants, conversions, colours, coordinates",
+  "eventsBasedBehaviorsCount": 0,
+  "eventsFunctionsCount": 13,
+  "description": "A general extention that adds additional math functions and constants expressions.\nThe extension includes:\n - Mathematical constants\n - Recursive functions like factorials\n - Coordinates conversion for the polar and isometric systems\n - And much more..."
```

Comment:
Avoid "much more", it's not useful. List the content instead.

`Additional math functions and constants expressions:\n - Mathematical constants\n - Greatest common factor/Lowest common multiple\n - Factorial\n - Coordinates conversion for polar and isometric systems`